start.sh

```diff
@@ -9,22 +9,35 @@ echo "|__/|__/\___/_.___/\____/_____/_/  /_/   "
 echo                          
 echo -e "\e[39m"
 
-python -c "import sys;ret = 1 if sys.version_info <= (3, 0) else 0;print('Checking python version... ' + ('3.x, good!' if ret == 0 else '2.x'));sys.exit(ret);"
-if [ $? -eq 1 ]; then
+almost_there(){
     echo 
     echo "===================="
     echo "You're almost there!"
     echo "===================="
+}
+
+# Check python version
+python -c "import sys;ret = 1 if sys.version_info <= (3, 0) else 0;print('Checking python version... ' + ('3.x, good!' if ret == 0 else '2.x'));sys.exit(ret);"
+if [ $? -eq 1 ]; then
+	almost_there
     echo -e "\e[33mYour system is currently using Python 2.x. You need to install or configure your system to use Python 3.x. Check out http://docs.python-guide.org/en/latest/dev/virtualenvs/ for information on how to setup Python 3.x alongside your Python 2.x install.\e[39m"
     echo
     exit
 fi
 
+# Check GDAL version
+python -c "import sys;import re;import subprocess;version = subprocess.Popen([\"gdalinfo\", \"--version\"], stdout=subprocess.PIPE).communicate()[0].decode().rstrip();ret = 0 if re.compile('^GDAL [2-9]\.[1-9]+').match(version) else 1; print('Checking GDAL version... ' + ('{}, excellent!'.format(version) if ret == 0 else version));sys.exit(ret);"
```

Comment:
Ah, mostly because I'm more familiar with Python and less with Bash. :) I also call python -c a few lines above for another check. Perhaps somebody could rewrite it in Bash.